diff --git a/node_modules/nuxt/dist/pages/runtime/plugins/router.js b/node_modules/nuxt/dist/pages/runtime/plugins/router.js
index ed2753e..1cf1124 100644
--- a/node_modules/nuxt/dist/pages/runtime/plugins/router.js
+++ b/node_modules/nuxt/dist/pages/runtime/plugins/router.js
@@ -195,6 +195,12 @@ const plugin = defineNuxtPlugin({
           }
         })));
       } else if (import.meta.server && to.fullPath !== initialURL && (to.redirectedFrom || !isSamePath(to.fullPath, initialURL))) {
+
+        console.log('@@@@@@@@@@@@@@')
+        console.log('initialURL:', initialURL)
+        console.log('to.redirectedFrom:', to.redirectedFrom)
+        console.log('to.fullPath:', to.fullPath)
+
         await nuxtApp.runWithContext(() => navigateTo(to.fullPath || "/"));
       }
     });
