diff --git a/node_modules/nuxt/dist/pages/runtime/plugins/router.js b/node_modules/nuxt/dist/pages/runtime/plugins/router.js
index ed2753e..da058dc 100644
--- a/node_modules/nuxt/dist/pages/runtime/plugins/router.js
+++ b/node_modules/nuxt/dist/pages/runtime/plugins/router.js
@@ -194,7 +194,7 @@ const plugin = defineNuxtPlugin({
             path: to.fullPath
           }
         })));
-      } else if (import.meta.server && to.fullPath !== initialURL && (to.redirectedFrom || !isSamePath(to.fullPath, initialURL))) {
+      } else if (import.meta.server && to.fullPath !== initialURL && (to.redirectedFrom || !isSamePath(to.fullPath, decodeURI(initialURL)))) {
         await nuxtApp.runWithContext(() => navigateTo(to.fullPath || "/"));
       }
     });
